pr_agent/tools/pr_description.py

```diff
@@ -74,10 +74,11 @@ async def run(self):
 
             await retry_with_fallback_models(self._prepare_prediction)
 
-            get_logger().info(f"Preparing answer {self.pr_id}")
+            get_logger().info("Preparing description regarding", extra={"pr_id": self.pr_id})
             if self.prediction:
                 self._prepare_data()
             else:
+                get_logger().error("Prediction failed to prepare", extra={"pr_id": self.pr_id})
```

Comment:
**Suggestion:** Include the PR ID within the log message to provide more context in the log output.
```suggestion
                get_logger().error(f"Prediction failed to prepare for PR {self.pr_id}", extra={"pr_id": self.pr_id})
```